cogs/fun.py

```diff
@@ -214,6 +215,39 @@ async def slot(self, ctx: Context[BotT]):
         else:
             await ctx.send(f"{slotmachine} No match, you lost 😢")
 
-
+    @commands.command()
+    async def dice(self,ctx:Context[BotT]):
+        """ Dice game. Good luck """
+        bot_dice=(random.randint(1, 6))
+        player_dice=(random.randint(1, 6))
+        await ctx.send(f"My dice   🎲 {bot_dice}\nYour dice 🎲 {player_dice}")
+        if bot_dice>player_dice:
+            await ctx.send('I won,try again!')
+        elif bot_dice==player_dice:
+            await ctx.send('Its a tie!')
+        else:
+            await ctx.send('Congrats,you won🎉!')
+
+    @commands.command(aliases=['colors'])
+    async def roula(self,ctx:Context[BotT]):
```

Comment:
Instead of using ctx.message.content that requires strippping the prefix, take usage of discord.py's built-in arguments for commands, an example would be the following:
```py
async def roula(self, ctx: Context[BotT], picked_colour: str):
   colors= ['blue','red','green','yellow']
   picked_colour = picked_colour.lower()
   if picked_colour not in colors:
      await ctx.send('Please give correct color')
```